ci-operator/config/openshift/cluster-api/openshift-cluster-api-master.yaml

```diff
@@ -35,12 +35,6 @@ tag_specification:
 tests:
 - as: unit
   commands: |
-    go get -u github.com/golang/dep/cmd/dep
```

Comment:
Don't we still need dep for the moment?

Reply:
good question, I don't see how it's being used anyhow though